docs/howto/custom-rootfs-multiple-targets.md

```diff
@@ -0,0 +1,314 @@
+
+## Prerequisites
+
+- A running self-hosted Landscape server version `24.10~beta.5` or later;
+- Multiple Windows machines already registered with Landscape via Ubuntu Pro for WSL;
+- Make sure you have installed `curl` and `jq`, if you're following this guide on a Linux machine.
+- Familiarity with Bash and/or PowerShell.
+
+## Prepare the environment
+
+To ease the subsequent commands, adjust and export the following environment variables:
+
+`````{tabs}
+
+````{group-tab} Bash
+```bash
+# Credentials to authenticate the API requests
+export LANDSCAPE_USER_EMAIL=admin@mib.com
+export LANDSCAPE_USER_PASSWORD=mib
+export LANDSCAPE_URL=https://landscape.mib.com
+
+# The URL of the custom rootfs to be deployed:
+export ROOTFS_URL="http://landscape.mib.com:9009/ubuntu-24.04-custom.tar.gz"
+
+# The list of IDs of the different Windows machines on which we are going to deploy WSL instances:
+export PARENT_COMPUTER_IDS=(26 30 31)
+
+# The name of the WSL instance to be created:
+export COMPUTER_NAME=Carbonizer
+
+# Path to the cloud-config file whose contents will be used to initialize the WSL instances:
+export CLOUD_INIT_FILE="~/Downloads/init.yaml"
+```
+
+````
+
+````{group-tab} PowerShell
+```powershell
+# Credentials to authenticate the API requests
+$LANDSCAPE_USER_EMAIL="admin@mib.com"
+$LANDSCAPE_USER_PASSWORD="mib"
+$LANDSCAPE_URL="https://landscape.mib.com"
+
+# The URL of the custom rootfs to be deployed:
+$ROOTFS_URL="http://landscape.mib.com:9009/ubuntu-24.04-custom.tar.gz"
+
+# The list of IDs of the different Windows machines on which we are going to deploy WSL instances:
+$PARENT_COMPUTER_IDS=@(26, 30, 31)
+
+# The name of the WSL instance to be created:
+$COMPUTER_NAME="Carbonizer"
+
+# Path to the cloud-config file whose contents will be used to initialize the WSL instances:
+$CLOUD_INIT_FILE="~/Downloads/init.yaml"
+```
+
+````
+
+`````
+
+
+Generate a Base64-encoded string with the cloud-config data:
+
+`````{tabs}
+
+````{group-tab} Bash
+
+```bash
+BASE64_ENCODED_CLOUD_INIT=$(cat $CLOUD_INIT_FILE | base64 --wrap=0)
+```
+
+````
+
+````{group-tab} PowerShell
+```powershell
+$content = Get-Content -Path $CLOUD_INIT_FILE -Raw
+$bytes = [System.Text.Encoding]::UTF8.GetBytes($content)
+$BASE64_ENCODED_CLOUD_INIT = [System.Convert]::ToBase64String($bytes)
+```
+
+````
+
+`````
+
+## Authenticate against the Landscape API
+
+Build the authentication payload of the form: `{"email": "admin@mib.com", "password": "mib"}` using the values exported in prior steps:
+
+`````{tabs}
+
+````{group-tab} Bash
+```bash
+LOGIN_JSON=$( jq -n \
+    --arg em "$LANDSCAPE_USER_EMAIL" \
+    --arg pwd "$LANDSCAPE_USER_PASSWORD" \
+    '{email: $em, password: $pwd}' )
+```
+````
+
+````{group-tab} PowerShell
+```powershell
+$LOGIN_JSON = @{
+ email = "$LANDSCAPE_USER_EMAIL"
+ password = "$LANDSCAPE_USER_PASSWORD"
+} | ConvertTo-Json
+```
+````
+`````
+
+Issue an authenticate request and retrieve the JWT to be used in the subsequent API requests.
```

Comment:
Will JWT be understood by all readers?

Maybe, but if in doubt define on first use:

> the JSON Web Token (JWT)